panml/llm/huggingface.py

```diff
@@ -77,13 +76,13 @@ def __init__(self, model: str, source: str, model_args: dict) -> None:
             if load_peft_lora:
                 # Set LoRA trained model
                 self.peft_config = PeftConfig.from_pretrained(self.model_name)
-                if 'flan' in self.model_name:
-                    self.model_hf = AutoModelForSeq2SeqLM.from_pretrained(self.peft_config.base_model_name_or_path, **model_args, local_files_only=True)
-                elif 'bert' in self.model_name:
-                    self.model_hf = AutoModelForMaskedLM.from_pretrained(self.peft_config.base_model_name_or_path, **model_args, local_files_only=True)
```

Comment:
are we not giving the option anymore to load local files here?

Reply:
Ok these are the comments

Reply:
ahh thanks - makese sense